common/config/presets/c_base_node.toml

```
@@ -182,6 +182,8 @@ track_reorgs = true
 # The maximum simultaneous comms RPC sessions allowed (default value = 100). Setting this to -1 will allow unlimited
 # sessions.
 #rpc_max_simultaneous_sessions = 100
+# The maximum comms RPC sessions allowed per peer (default value = 10).
+#rpc_max_sessions_per_peer = 11
```

```suggestion
#rpc_max_sessions_per_peer = 10
```
Is there a reason why the commented-out value is not the default?

---

Thanks - just a mistake, had it as one more than the maximum RPC pool size in the wallet, but setting it at 10 should be OK.